index.html

```diff
@@ -621,10 +623,30 @@ <h4>issuance</h4>
 
       <section>
         <h3>Instead of...</h3>
-        <p class="issue">This section needs to be defined.</p>
+        <p>
+          This representation can be obtained by:
```

Comment:
To be clear, there are LOTs of ways you can produce a JWT for this "path"... the goal here was to describe just one simple way, that relates this path to the concrete production rules above.

Reply:
```suggestion
          The following steps are one way to obtain this representation:
```